config/model_configs/amip_diagedmf.yml

```diff
@@ -0,0 +1,15 @@
+anim: false
+atmos_config_file: "config/gpu_configs/gpu_aquaplanet_diagedmf.yml"
+dt_cpl: 100
+dt_save_state_to_disk: "12hours"
```

Comment:
This is 12hours for the AMIP setups, and Inf for the aquaplanet setups. Which of these do you think we would want? @LenkaNovak

Reply:
Thank you for such clear instructions! I added a new flag `use_coupler_diagnostics` and it's used in the driver to toggle the diagnostics